### PR TITLE
Move leanblueprint jobs to regular workflow

### PR DESCRIPTION
**Pull request by MithicSpirit**
We want it to work even if there are minor (nonbreaking) issues in the
commit.

### CODE REVIEW

**Comment by MAO3J1m0Op** [comment]
Works per my testing

File: .github/workflows/build-strict.yml [hunk 1]
```diff
@@ -51,64 +45,3 @@ jobs:
 
       - name: build project
         run: lake build --wfail
-
-      - name: Build blueprint and copy to `docs/blueprint`
-        uses: xu-cheng/texlive-action@v2
-        with:
-          docker_image: ghcr.io/xu-cheng/texlive-full:20231201
```
Comment (MAO3J1m0Op) [inline .github/workflows/build-strict.yml:58]: Maybe if we upgrade to the latest docker image by removing this line, it’ll be stored locally and won’t have to be rebuilt every time.

**Review by MAO3J1m0Op — APPROVED** [review]

**Review by william11339 — APPROVED** [review]

File: .github/workflows/build-strict.yml (hunk 1 above)
Comment (MithicSpirit) [reply .github/workflows/build-strict.yml:58]: That line is a parameter for how the `xu-cheng/texlive-action@v2` action works, and it doesn't seem to do any form of caching. I'll experiment with not using that action.

Comment (MithicSpirit) [reply .github/workflows/build-strict.yml:58]: This is something for us to maybe investigate in the future.

**Review by Anoushka1406 — APPROVED** [review]
lgtm

**Pull request merged**